### PR TITLE
Migrate to Swift 3 syntax

### PR DESCRIPTION
**Pull request by dewski**
I was trying to use this in a Swift 3 project with no success, luckily most of this translated 1:1 over to Swift 3 code and its idioms.

image

Feel free to pick and choose any of these changes!

/cc @cezarywojcik

### CODE REVIEW

File: CWStatusBarNotification/CWStatusBarNotification/CWStatusBarNotification.swift [hunk 1]
```diff
@@ -359,7 +359,7 @@ public class CWStatusBarNotification : NSObject {
 
     // MARK: - display notification
 
-    public func displayNotification(withMessage message : String,
+    public func displayNotification(_ message : String,
```
Comment (cezarywojcik) [inline CWStatusBarNotification/CWStatusBarNotification/CWStatusBarNotification.swift]: Please keep the `withMessage` label to retain parity with obj-c api

File: CWStatusBarNotification/CWStatusBarNotification/CWStatusBarNotification.swift [hunk 2]
```diff
@@ -11,515 +11,510 @@ import UIKit
...
+        guard let label = self.notificationLabel else {
+            return
+        }
+        self.notificationWindow?.rootViewController?.view.addSubview(label)
+        self.notificationWindow?.rootViewController?.view.bringSubview(toFront: label)
+        self.notificationWindow?.isHidden = false
+
+        // checking for screen orientation change
+        NotificationCenter.default.addObserver(self,
+                                                         selector: #selector(CWStatusBarNotification.updateStatusBarFrame),
```
Comment (cezarywojcik) [inline CWStatusBarNotification/CWStatusBarNotification/CWStatusBarNotification.swift]: what's going on with indentation here?

File: CWStatusBarNotification/CWStatusBarNotification/CWStatusBarNotification.swift [hunk 3]
```diff
@@ -11,515 +11,510 @@ import UIKit
...
+            view.removeFromSuperview()
+            self.statusBarView?.removeFromSuperview()
+            self.notificationWindow?.isHidden = true
+            self.notificationWindow = nil
+            self.customView = nil
+            self.notificationLabel = nil
+            self.notificationIsShowing = false
+            self.isNotificationDismissing = false
+            NotificationCenter.default.removeObserver(self,
+                                                                name: .UIApplicationDidChangeStatusBarFrame,
```
Comment (cezarywojcik) [inline CWStatusBarNotification/CWStatusBarNotification/CWStatusBarNotification.swift]: this intentation is weird too

**Review by cezarywojcik — CHANGES_REQUESTED** [review]

**Comment by JeromeTW** [comment]
Is this brunch work for Swift 3 and Xcode8？？
how to install it by cocoapod?
pod 'CWStatusBarNotification', :git => 'https://github.com/cezarywojcik/CWStatusBarNotification', :commit => 'f2a358dc8b7db9e6136f3a993a08b49e56cca5fc'

**Pull request closed without merging**